pages/templates/partials/initiative_endorsement_old.html

```diff
@@ -10,7 +10,7 @@
             <div class="row">
                 <div class="container">
                     <div class="page-sub-nav">
-                        <a href="{% pageurl page.get_parent %}"><span class="glyphicon glyphicon-chevron-left"></span></a> Our Initiatives / {{ initiative.state }} / {{ initiative.title }}: {{ initiative.name }}
+                        <a href="{% pageurl page.get_parent %}"><span class="glyphicon glyphicon-chevron-left"></span></a> Our Initiatives / {{ initiative.state }} / {{ initiative.vote | yesno | title }} on {{ initiative.title }}: {{ initiative.name }}
```

Comment:
why not use the new property you created?

Reply:
👍